### ISSUE TITLE
Focus ring around the checkbox control should be visible on select/deselect with keyboard when NVDA is ON

### ISSUE THREAD

**Issue by RashmiBal** [microsoft/fluentui#18857]
### Environment Information

- **Package version(s)**: https://developer.microsoft.com/en-us/fluentui#/controls/web/checkbox
- **Browser and OS versions**: Chrome Version 91.0.4472.124 (Official Build) (64-bit) + Windows 10 + Screenreader NVDA Version 2020.3

### Describe the issue:

1. Go to https://developer.microsoft.com/en-us/fluentui#/controls/web/checkbox on Chrome Browser 
2. TAB to move focus to one of the checkbox in the sample
3. Hit SPACEBAR to select or deselect the checkbox.
4. See Focus ring around the checkbox control (GOOD)
5. Now, turn ON NVDA screenreader
6. Assuming you have focus on the checkbox in the sample (step 2 above)
7. Hit SPACEBAR to select or deselect the checkbox.

### Please provide a reproduction of the issue in a codepen:
Use the sample on this page https://developer.microsoft.com/en-us/fluentui#/controls/web/checkbox

#### Actual behavior:

Focus ring around the checkbox control goes missing (BAD), when NVDA is turned ON.
====> https://media.giphy.com/media/8lNvaC5PPeaOvSXS0j/giphy.gif

#### Expected behavior:

Focus ring around the checkbox control should be visible even when the NVDA or any screenreader is turned ON

**Comment by ling1726** [comment]
Confirm that I can repro with NVDA. This focus ring will disappear only after checking/unchecking a checkbox.

@smhigley can you find an owner for this in in the v8 team ?
